sources/imperative/ReaderUtils.swift

```diff
@@ -0,0 +1,37 @@
+import Foundation
+
+extension InputStream {
+    /// Stream for reading from stdin.
+    public static var stdin: InputStream {
```

Comment:
What file should this extension be in?

Reply:
TIL I learned you can't have the same file name in the same target. I originally tried naming this file `Utils.swift`.

```
$ swift build
<unknown>:0: error: filename "Utils.swift" used twice: 'sources/declarative/Utils.swift' and 'sources/i<unknown>:0: note: filenames are used to distinguish private declarations with the same name
```